src/java.base/share/classes/jdk/internal/classfile/impl/StackMapDecoder.java

```diff
@@ -253,22 +296,17 @@ List<StackMapFrameInfo> entries() {
                     stack = List.of(newStack);
                 }
             }
-            Label label = ctx.getLabel(bci);
-            if (!deferredUnsetFields.isEmpty()) {
-                // technically we only have one assert at once, just in case
-                // of duplicate asserts...
-                for (var deferredList : deferredUnsetFields) {
-                    entries.add(new StackMapFrameImpl(ASSERT_UNSET_FIELDS,
-                                label, oldLocals, oldStack, deferredList));
-                }
-                deferredUnsetFields.clear();
+            if (actualFrameType != 247 && !unsetFields.isEmpty() && !locals.contains(SimpleVerificationTypeInfo.UNINITIALIZED_THIS)) {
```

Comment:
we should use a constant not a literal, 247, here

Reply:
Very true, this literal is also wrong